opa/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "display_name": "opa",
+  "maintainer": "ara.pulido@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "opa",
+  "metric_prefix": "opa.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "OPA integration",
+  "guid": "a68c72bd-a16a-4fcf-8911-43575dd722b9",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
```

Comment:
(question) Are they all supported?

Reply:
Actually, no, I will remove mac and win :)